functions/lib/cla.js

```diff
@@ -1,74 +1,138 @@
 module.exports = Cla
 
+/**
+ * Returns CLA-related functions.
+ * @param db {FirebaseFirestore.Firestore}
+ */
 function Cla (db) {
-  async function isClaSigned (user, ref) {
-    if (!user || !user.email) {
-      console.log('email is not provided')
+  /**
+   * Returns a string that uniquely identifies the given identity object.
+   * @param identity {{type: string, value: string}}
+   * @returns {string|boolean}
+   */
+  function identityKey (identity) {
+    if (!identity) {
       return false
     }
+    if (!Object.prototype.hasOwnProperty.call(identity, 'type') ||
```

Comment:
minor comment, we can use `!('type' in identity)` to make the code more concise

Reply:
done. much better.